cdap-api/src/main/java/co/cask/cdap/api/dataset/table/Tables.java

```diff
@@ -89,7 +89,7 @@ public static void createTable(ApplicationConfigurer configurer, String datasetN
   /**
    * Creates properties for {@link Table} or {@link Table} data set instance.
    * @param level level on which to detect conflicts in changes made by different transactions
-   * @param ttl time to live for data written into a table, in ms. {@link #NO_TTL} means unlimited
+   * @param ttl time to live for data written into a table, in seconds. {@link #NO_TTL} means unlimited
```

Comment:
this is more like a documentation bug, it should be seconds instead of ms. also i am not sure how this Class is used, i don't see any usages of this.. i see this is part of cdap-api, but not sure how it can use used..